mavis/assemble.py

```diff
@@ -389,11 +390,14 @@ def assemble(
     assembly.trim_noncutting_paths_by_freq(min_edge_trim_weight)
 
     path_scores = {}
+    for component in sorted([i for i in digraph_connected_components(assembly)]):
```

Comment:
why the i for i in ? are you just trying to cast to a list? does digraph_connected_components not return a list? if so maybe just use 

```
list(digraph_connected_components(assembly))
```

Reply:
ah yes I can just do that